packages/node_modules/@webex/redux-module-conversation/src/actions.js

```diff
@@ -438,7 +443,8 @@ export function handleAdaptiveCardSubmitAction(url, actionInput, parentId, spark
       {url},
       actionInput,
       {id: parentId}
-    ).then((activity) => {
-      dispatch(acknowledgeAdaptiveCardSubmitAction(activity));
-    });
+    ).then((activity) =>
+      dispatch(acknowledgeAdaptiveCardSubmitAction(activity.id, parentId)))
+      .catch(() =>
+        dispatch(acknowledgeAdaptiveCardSubmitAction({activityId: null, parentId: null})));
```

Comment:
The `acknowledgeAdaptiveCardSubmitAction(activityId, parentId)` function accepts two params, not a single param object. 

```suggestion
        dispatch(acknowledgeAdaptiveCardSubmitAction(null, null)));
```

Reply:
My Bad , I passed it as an object. I changed to two params